rasterio/gdal.pxi

```diff
@@ -361,7 +361,8 @@ cdef extern from "gdal.h" nogil:
     int GDALRasterIO(GDALRasterBandH band, int, int xoff, int yoff, int xsize,
                      int ysize, void *buffer, int width, int height, int,
                      int poff, int loff)
-
+    CPLErr GDALGetRasterStatistics(GDALRasterBandH band, int approx, int force, double *min, double*max, double *mean, double *std)
+    void GDALDatasetClearStatistics(GDALDatasetH hDS)
```

Comment:
I believe `IF (CTE_GDAL_MAJOR_VERSION, CTE_GDAL_MINOR_VERSION) >= (3, 2):` is needed here.

Reply:
I'm not sure... the tests didn't fail on this.

Reply:
🤔 that's strange ... well, if it ain't broke ...